### PR TITLE
Added animations and grunt

### PR DESCRIPTION
**Pull request by rafinskipg**
Hi buddy,  

I added animate.css for the page transitions and grunt task for running the enviroment ;)

Run 

```
npm install
```

Then you can do 

```
grunt
```

:) Enjoy!

### CODE REVIEW

**Comment by viniciusalmeida** [comment]
hi @rafinskipg, these update is very interesting.
thanks a lot for your contribution.

coming soon i go verify your commits.

thanks again!

**Comment by rafinskipg** [comment]
No problemo, if you like it i will contribute more updates , like up/down animations, circle animations like reveal.js . Also if you find something wrong, tell me

File: index.html [hunk 1]
```diff
@@ -4,11 +4,13 @@
     <meta charset='utf-8'>
     <title>ninjaPresentation</title>
     <script src='//cdnjs.cloudflare.com/ajax/libs/polymer/0.0.20130816/polymer.min.js'></script>
+    <!--<script src='/bower_components/polymer/polymer.min.js'></script>-->
```
Comment (viniciusalmeida) [inline index.html]: Hi @rafinskipg!
I have my questions if is important include polymer on bower components.
I've been using the CDN to have few dependencies. Why you decide for this?

Comment (rafinskipg) [inline index.html]: Hi, for offline development, also if you want to distribute this component, it has to include all the dependencies. 

I really don't know how webcomponents are distributed, by the way..

**Comment by viniciusalmeida** [comment]
your implementation of animations is very interesting @rafinskipg . i accept your pull request.
but, i will propose some modifications:
- the comment with script of polymer in bower components should be deleted of `index.html`
- on `Gruntfile.js`, bower not need be cited
- the prefixes `-moz` and `-o` aren't necessary. the ninja-presentation don't are enable on Firefox or Opera because these browsers no supports HTML templates yet

replying your question, the principal channel is the customelements.io. At least for ninja-presentation

thanks

**Comment by rafinskipg** [comment]
Ok, viniciusalmeida, check if it is ok this way,

I removed all the css that wasn't used. We can add more later, it's from https://daneden.me/animate/
Thanks

**Pull request merged**